content/en/error_tracking/frontend/mobile/reactnative.md

```diff
@@ -5,4 +5,914 @@ code_lang: reactnative
+
+{{% /collapse-content %}}
+
+{{% collapse-content title="Automate the upload on each build" level="h5" %}}
+
+Install the plugin like in the previous step.
+
+Find the loop on `applicationVariants` in the `android/app/build.gradle` file. It should look like `applicationVariants.all { variant ->`.
+
+Inside the loop, add the following snippet:
+
+```groovy
+        if (project.tasks.findByName("minify${variant.name.capitalize()}WithR8")) {
+            tasks["minify${variant.name.capitalize()}WithR8"].finalizedBy { tasks["uploadMapping${variant.name.capitalize()}"] }
+        }
+```
+
+**Note**: Re-uploading a source map does not override the existing one if the version has not changed.
+
+{{% /collapse-content %}}
+
+
+## Troubleshooting
+
+### Usage with `use_frameworks!`
+
+If you have `use_frameworks!` enabled in your `Podfile`, running `pod install` after adding the SDK is likely to trigger an error like this one:
+
+```shell
+The 'Pods-MyApp' target has transitive dependencies that include statically linked binaries: (DatadogSDKBridge, DatadogSDKCrashReporting)
+```
+
+To prevent that error, edit your `Podfile` to install the React Native SDK pod as a static library:
+
+```ruby
+static_libraries = ['DatadogSDKReactNative']
+
+# Turn pods with static dependencies into static libraries by overriding the static_framework? function to return true
+pre_install do |installer|
+  installer.pod_targets.each do |pod|
+    if static_libraries.include?(pod.name)
+      def pod.static_framework?;
+        true
+      end
+      def pod.build_type;
+        Pod::BuildType.static_library
+      end
+    end
+  end
+end
+```
+
+**Note**: This solution comes from this [StackOverflow][8] post.
+
+
+## Further Reading
+
+{{< partial name="whats-next/whats-next.html" >}}
+
+[1]: /error_tracking/
+[2]: https://microsoft.github.io/code-push/
+[3]: /real_user_monitoring/mobile_and_tv_monitoring/react_native/setup/codepush
+[4]: https://docs.expo.dev/
+[5]: /real_user_monitoring/mobile_and_tv_monitoring/react_native/setup/expo
+[6]: /account_management/api-app-keys/#api-keys
+[7]: /account_management/api-app-keys/#client-tokens
+[8]: https://github.com/DataDog/datadog-react-native-wizard
+[9]: https://app.datadoghq.com/source-code/setup/rum
+[10]: https://github.com/DataDog/react-native-performance-limite
+[11]: https://github.com/DataDog/datadog-ci/tree/master/src/commands/react-native#xcode
+[12]: /real_user_monitoring/mobile_and_tv_monitoring/ios/error_tracking/?tabs=cocoapods#symbolicate-crash-reports
+[13]: https://reactnative.dev/docs/signed-apk-android#enabling-proguard-to-reduce-the-size-of-the-apk-optional
+[14]: https://plugins.gradle.org/plugin/com.datadoghq.dd-sdk-android-gradle-plugin
+[15]: https://github.com/DataDog/dd-sdk-android-gradle-plugin
+[16]: https://app.datadoghq.com/rum/application/create
+[17]: /real_user_monitoring/mobile_and_tv_monitoring/react_native/
```

Comment:
```suggestion
[17]: /real_user_monitoring/application_monitoring/react_native/setup/

```

Reply:
It looks like this is the link for React Native setup documentation -- could you please confirm?